### PR TITLE
Device class is now more testable

### PR DESCRIPTION
**Pull request by f-meloni**
Device class is now more easy to mock.
What i did to accomplish that is just remove all the static variables from the class.
I also made the class more swift like.
Another good point is that now the `InformationsTableViewController` has his own device instance, and this makes it more easy to test, with a snapshot test for example, but also with UI tests, with mocked `Device` class

### CODE REVIEW

File: Framework/Dotzu/Dotzu/Device.swift [hunk 1]
```diff
@@ -9,19 +9,27 @@
 import UIKit
 
 struct Device {
-
     var osVersion: String?
-    static var screenResolution: String? {
+    var screenResolution: String?
+    let deviceModel: DeviceModel = DeviceModel.current
```
Comment (f-meloni) [inline Framework/Dotzu/Dotzu/Device.swift]: `deviceModel` doesn't need to be a `var`

**Review by remirobert — APPROVED** [review]
LGTM

File: Framework/Dotzu/Dotzu/Device.swift (hunk 1 above)
Comment (remirobert) [reply Framework/Dotzu/Dotzu/Device.swift]: because it's optional maybe ?

File: Framework/Dotzu/Dotzu/Device.swift [hunk 2]
```diff
@@ -9,19 +9,27 @@
...
+    }
+    
+    private func deviceScreenResolution() -> String {
         let scale = UIScreen.main.scale
         let dimension = UIScreen.main.bounds
         return "\(dimension.size.width*scale)*\(dimension.size.height*scale)"
     }
-    static var deviceModel: DeviceModel = DeviceModel.current
-
-    var getScreenSize: String?
```
Comment (f-meloni) [inline Framework/Dotzu/Dotzu/Device.swift:21]: This was not used, then i removed it.

Comment (remirobert) [reply Framework/Dotzu/Dotzu/Device.swift:21]: good spot 👍

**Review by remirobert — COMMENTED** [review]
Very good point

**Comment by CocoaDebug** [comment]
Hi, I released [DebugMan](https://github.com/liman123/DebugMan), it is forked from [Dotzu](https://github.com/remirobert/Dotzu).
Welcome to use! 😃 thanks!
@f-meloni

**Pull request merged**